crates/config/src/lib.rs

```diff
@@ -2212,7 +2212,7 @@ impl Default for Config {
             allow_paths: vec![],
             include_paths: vec![],
             force: false,
-            evm_version: EvmVersion::default(),
+            evm_version: EvmVersion::Paris,
```

Comment:
Any reason we default to Paris? what does upstream do?

Reply:
upstream defaults to `Paris` actually, we had a regression in defaulting to `default` that I have identified as well